server-spi/src/main/java/org/keycloak/models/ThemeManager.java

```diff
@@ -26,6 +26,17 @@ public interface ThemeManager {
      */
     Theme getTheme(String name, Theme.Type type) throws IOException;
 
+    /**
+     * Returns the specified theme for the specified type.
+     *
+     * @param name
+     * @param type
+     * @param fallbackToDefaultTheme if true (the default) and theme is not found, this method will return built-in default theme instead. If false and theme is not found, this method will return null
+     * @return
+     * @throws IOException
+     */
+    Theme getTheme(String name, Theme.Type type, boolean fallbackToDefaultTheme) throws IOException;
```

Comment:
See below, but thinking we could do without this overloaded method, especially since it's only used in a single place.